lib/iex/lib/iex/evaluator.ex

```diff
@@ -24,6 +24,21 @@ defmodule IEx.Evaluator do
     end
   end
 
+  @doc """
+  Gets a value out of the binding, using the provided
+  variable name and map key path.
+  """
+  @spec value_from_binding(pid, atom, [atom]) :: {:ok, any} | :error
+  def value_from_binding(evaluator, var_name, map_key_path) do
+    send evaluator, {:value_from_binding, self(), var_name, map_key_path}
```

Comment:
Thanks, I often use refs when messaging but forgot to do so here.  I'm glad, though; it gave you the opportunity to teach me some things about it that I didn't know, like the VM optimization :).

Anyhow, fixed.